.php_cs

```diff
@@ -39,15 +39,16 @@ return PhpCsFixer\Config::create()
             'align_equals' => true,
         ],
         'blank_line_after_opening_tag' => true,
-        'blank_line_before_return' => true,
+        'blank_line_before_statement' => true,
```

Comment:
Good catch, I'll probably change the default later (the default is [ "break", "continue", "declare", "return", "throw", "try" ]) to remove the break and throws because I feel it's to much line space now ;)